content/webhooks.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Webhooks
+body_id: webhooks
+---
+If you have created an [OAuth Application](/oauth-integration/#creating-an-oauth-application), you will have the ability to create webhooks. Webhooks can be defined whilst creating/editing your OAuth application and multiple URLs can be defined by separating with a space.
+
+### Usage
+
+Webhooks can be used as an alternative to polling endpoints to fetch updates for pages. It is important that webhook receiving endpoints return a 200 response as soon as possible, as requests will timeout after 2000ms. It is recommended not to perform any work within the receive request, but to queue the data to be processed at a later stage, so the request can be exited early.
+
+To respond as quickly to a webhook, subscribers should defer any processing of the notification until after the response is done. You may do this in a few ways, but two popular options are to spawn a new thread to handle the processing, or to place the notification in a queue for later processing. Since you're likely already using queues for your processing, this is probably your best option.
```

Comment:
"To respond as quickly to a webhook, subscribers should defer any processing of the notification until after the response is done." 
As quickly as what? I know I'm arguing sentence structure but this phrasing is confusing.